clang/test/Driver/sycl-host-compiler-old-model.cpp

```diff
@@ -76,3 +76,17 @@
 // CHECK-ZC-CPLUSPLUS: "/Zc:__cplusplus"
 // CHECK-ZC-CPLUSPLUS-MINUS: "/Zc:__cplusplus-"
 // CHECK-NO-ZC-CPLUSPLUS-NOT: "/Zc:__cplusplus"
+
+/// -fsycl-host-compiler -save-temps behavior
+// RUN: %clangxx -### -fsycl-host-compiler=g++ -fsycl --no-offload-new-driver \
+// RUN:          -save-temps -c %s 2>&1 \
```

Comment:
Is `-save-temps` and `--save-temps` the same?

Reply:
Yes, they are equivalent.  The option is formed with both '-' and '--' as valid option specifiers.